apps/console/src/features/organization-discovery/components/add-organization-discovery-domains.tsx

```diff
@@ -90,17 +91,35 @@ const FORM_ID: string = "edit-organization-email-domains-form";
 const AddOrganizationDiscoveryDomains: FunctionComponent<AddOrganizationDiscoveryDomainsPropsInterface> = (
     props: AddOrganizationDiscoveryDomainsPropsInterface
 ): ReactElement => {
-    const { ["data-componentid"]: componentId } = props;
+    const { 
+        isReadOnly,
+        ["data-componentid"]: componentId
+    } = props;
 
     const { t } = useTranslation();
 
     const dispatch: Dispatch = useDispatch();
 
+    const allowedScopes: string = useSelector((state: AppState) => state?.auth?.allowedScopes);
+    const featureConfig: FeatureConfigInterface = useSelector((state: AppState) => state.config.ui.features);
```

Comment:
Shall we add nullable checks

Reply:
Grab the feature specific config instead:

Ref: https://github.com/wso2/identity-apps/blob/9566504a9270c7805af809ea1087a95d53d338fb/apps/console/src/features/organization-discovery/pages/assign-organization-discovery-domains-page.tsx#L50

Reply:
Fixed with f14b8e1c35cbf764611d52797b803c6294e0fc0f